.github/workflows/upload-compilation-result.yaml

```diff
@@ -49,6 +50,14 @@ jobs:
           body="${body//$'\n'/'%0A'}"
           body="${body//$'\r'/'%0D'}"
           echo "::set-output name=result::${body}"
+      - id: test-personas
+        run: |
+          # Test the personas bilans against production
+          body="$(yarn test 2> /dev/null| tail --lines=+3 | head --lines=-1)"
+          body="${body//'%'/'%25'}"
```

Comment:
Je n'ai jamais écrit de github workflow, cette syntaxe est assez cryptique, c'est inévitable ?

Reply:
Il y a probablement mieux, mais je voulais garder la syntaxe que vous utilisez pour le reste de votre workflow, pour ne pas introduire trop de changement